modules/swagger-codegen/src/main/resources/python/__init__package.mustache

```diff
@@ -1,16 +1,17 @@
 # coding: utf-8
 
+# flake8: noqa
+
 {{>partial_header}}
 
 from __future__ import absolute_import
 
-# import models into sdk package
-{{#models}}{{#model}}from .models.{{classFilename}} import {{classname}}
```

Comment:
hardcoded `models` and `apis` were overriding the values specified by user during configuration.